pypeit/deprecated/combine.py

```diff
@@ -0,0 +1,204 @@
+
+# TODO JFH This code is a giant piece of crapp that should be rewritten from scratch
```

Comment:
JFH should remove this insulting TODO, especially since
it has been deprecated.